src/Microsoft.AspNetCore.Http.Features/IFormFileCollection.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.AspNetCore.Http
     public interface IFormFileCollection : IReadOnlyList<IFormFile>
     {
         /// <summary>
-        /// Gets the file with the specified name.
+        /// Gets the first file in a collection that satisfies a specified name.
```

Comment:
What do you think of:
> Gets the file with the specified name. If there are multiple files with this name then the first match is returned.

Reply:
Yes, thanks. That would be better. I'll change